locust/runners.py

```diff
@@ -410,11 +414,15 @@ def start(self, locust_count, hatch_rate):
         
         self.state = STATE_HATCHING
 
-    def stop(self):
-        self.state = STATE_STOPPING
-        for client in self.clients.all:
-            self.server.send_to_client(Message("stop", None, client.id))
-        self.environment.events.test_stop.fire(environment=self.environment)
+    def stop(self, info=None):
```

Comment:
I don't think we should add the info parameter to the `stop` method. Instead we should log the message where we make the call to `stop()`.